apollo-router/src/axum_http_server_factory.rs

```diff
@@ -109,7 +110,13 @@ where
     RF: SupergraphServiceFactory,
 {
     ensure_listenaddrs_consistency(configuration, &endpoints)?;
-    if configuration.sandbox.enabled && !sandbox_on_main_endpoint(configuration) {
+    if configuration.sandbox.enabled
+        && !on_supergraph_endpoint(
+            configuration,
+            &configuration.sandbox.listen,
+            &configuration.sandbox.path,
+        )
+    {
         endpoints.insert(
```

Comment:
NIT This block looks like two instances of very similar code. Can it be factored out into a function?

Reply:
We decided to keep the homepage on / for now, so this code isn't duplicated anymore.